web-src/src/helpers/FormatHelper.js

```diff
@@ -68,3 +68,16 @@ export function newGroupingLabelGenerator() {
     return null;
   };
 }
+
+export function getErrorCodeSubstring(str) {
+  const result = str?.match(/{{(.*?)}}/);
+  return result ? result[1] : undefined;
+}
+
+export function createToastErrorMessage(message, localizedError) {
```

Comment:
Should we name this method more abstract, like getMessageWithLocalization or handleLocalizedResponse?
Please add a description to this method to explain its purpose.